api/types/stats.go

```diff
@@ -89,10 +89,15 @@ type NetworkStats struct {
 
 // Stats is Ultimate struct aggregating all types of stats of one container
 type Stats struct {
-	Read        time.Time    `json:"read"`
-	Network     NetworkStats `json:"network,omitempty"`
-	PreCPUStats CPUStats     `json:"precpu_stats,omitempty"`
-	CPUStats    CPUStats     `json:"cpu_stats,omitempty"`
-	MemoryStats MemoryStats  `json:"memory_stats,omitempty"`
-	BlkioStats  BlkioStats   `json:"blkio_stats,omitempty"`
+	Read time.Time `json:"read"`
+
+	// Network is for fallback stats where API Version <= 1.21
+	Network NetworkStats `json:"network,omitempty"`
```

Comment:
Updated.  I tried to not make a over deep change of the existing code. So I place the version-check the moment before this API return:
- If request version < 1.21, sum up those original data and return
- if request version >=1.21, original data will be return

But it still seems quite redundant. any better solutions please @calavera ?